javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,194 @@
+//freecodecamp user name : https://www.freecodecamp.org/wencho  
+//excercise 1
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+function removeMatch(array, itemToRemove) {
+    for (var i = 0; i < array.length - 1; i++) {
+        if (array[i] === itemToRemove) {
+            array.splice(i, 1);
+        }
+    }
+
+}
+
+removeMatch(names, nameToRemove);
+
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+
+
+//Excercise 2
+
+function timeToDestination(info) {
+    let hours = Math.floor(info.destinationDistance / info.speed);
+    let minutes = Math.floor(60 * (info.destinationDistance / info.speed - hours));
+    return hours + " hours and " + minutes + " minutes";
+}
+
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+const travelTime = timeToDestination(travelInformation);
+console.log(travelTime); // 4 hours and 42 minutes
+
+
+
+
+//Excercise 3
+
+
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "Sopranos",
+        days: 3,
+        hours: 14,
+        minutes: 0,
+    },
+    {
+        title: "Mr. Robot",
+        days: 1,
+        hours: 8,
+        minutes: 15,
+    },
+    {
+        title: "Friends",
+        days: 4,
+        hours: 2,
+        minutes: 20,
+    },
+    {
+        title: "How I met your mother",
+        days: 3,
+        hours: 4,
+        minutes: 16,
+    },
+    {
+        title: "Dark",  
+        days: 0,
+        hours: 22,
+        minutes: 58,
+    },
+    {
+        title: "Breaking bad",
+        days: 1,
+        hours: 22,
+        minutes: 30,
+    },
+];
+
+
+function logOutSeriesText(seriesList, lifeSpanInYears) {
+    let percentageArray = [];
+    let lifeSpanInMinutes = lifeSpanInYears * 365 * 24 * 60;
+    for (let i = 0; i < seriesList.length; i++) {
+        let lengthInMinutes = seriesList[i].days * 24 * 60 + seriesList[i].hours * 60 + seriesList[i].minutes;
+        let percentage = lengthInMinutes / lifeSpanInMinutes * 100;
+        percentageArray.push(percentage);
+        console.log(seriesDurations[i].title + " took " + percentage.toFixed(3) + " of my life");
+    }
+    let totalPercentage = 0;
+    for (let j = 0; j < percentageArray.length - 1; j++) {
```

Comment:
should it be like this ? i don't follow why `-1`
```suggestion
    for (let j = 0; j < percentageArray.length ; j++) {
```